src/Bicep.LangServer/BicepCompilationManager.cs

```diff
@@ -42,11 +45,19 @@ public BicepCompilationManager(ILanguageServerFacade server, ICompilationProvide
             this.scheduler = scheduler;
         }
 
-        public void RefreshCompilation(DocumentUri documentUri)
+        public void RefreshCompilation(DocumentUri documentUri, bool reloadBicepConfig = false)
         {
             var compilationContext = this.GetCompilation(documentUri);
+
             if (compilationContext is null)
             {
+                if (reloadBicepConfig &&
+                    workspace.TryGetSourceFile(documentUri.ToUri(), out ISourceFile? sourceFile) &&
+                    sourceFile is BicepFile)
+                {
+                    UpsertCompilationInternal(documentUri, null, sourceFile, reloadBicepConfig);
+                }
+
                 // the compilation we are refreshing no longer exists
```

Comment:
>exists [](http://example.com/codeflow?start=63&length=6)

The comment should be moved up above the additions.

Reply:
Moved the comment as part of this commit:
https://github.com/Azure/bicep/commit/1ee10711ad7bdd7412967c1ce6ae6f6a1923fa5b